.env.example

```
@@ -0,0 +1,6 @@
+LLM_API_KEY=
+LLM_BASE_URL=https://api.gapgpt.app/v1
+LLM_MODEL=gpt-5o
+AGENT_WORKDIR=.
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AGENT_WORKDIR key should go before the LLM_API_KEY key

(UnorderedKey)